pkgs/top-level/lua-packages.nix

```diff
@@ -3,134 +3,141 @@
    them.  Also, because most Nix expressions for Lua packages are
    trivial, most are actually defined here.  I.e. there's no function
    for each package in a separate file: the call to the function would
-   be almost as must code as the function itself. */
+   be almost as must code as the function itself.
+*/
 
-{ fetchurl, stdenv, lua, unzip, pkg-config
-, pcre, oniguruma, gnulib, tre, glibc, sqlite, openssl, expat
-, autoreconfHook, gnum4
-, postgresql, cyrus_sasl
-, fetchFromGitHub, which, writeText
-, pkgs
-, lib
-}@args:
+{ fetchurl, stdenv, lua, unzip, pkg-config, pcre, oniguruma, gnulib, tre, glibc
+, sqlite, openssl, expat, autoreconfHook, gnum4, postgresql, cyrus_sasl
+, fetchFromGitHub, which, writeText, pkgs, lib }@args:
 
 let
-  packages = ( self:
```

Comment:
why the change in indentation ? it makes reviewing harder

Reply:
My bad, ran `nixfmt` on it and didn't check the diff